lib/Request.js

```diff
@@ -88,6 +89,24 @@ Request.prototype.getParam = function(param) {
     return this._params.hasOwnProperty(param) ? this._params[param] : undefined;
 };
 
+Request.prototype.setQueries = function(queries) {
+    this._url.query = queries;
+    return this;
+};
+
+Request.prototype.getQueries = function() {
+    return this._url.query;
+};
+
+Request.prototype.setQuery = function(query, value) {
+    this._url.query[query] = value;
+    return this;
+};
+
+Request.prototype.getQuery = function(query) {
+    return this._url.query.hasOwnProperty(query) ? this._url.query[query] : undefined;
+};
+
```

Comment:
getters and setters could work for the `setQueries` and `getQueries` methods, but the interface is not compatible with `getQuery(name)` and `setQuery(name, value)` unless we promote `this._url.query` to a new object.

Either way, the public interface doesn't change, but this implementation probably requires the least amount of code.